package.json

```diff
@@ -73,5 +73,8 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "devDependencies": {
```

Comment:
This should be a regular dependency at least for how I think about things. Gatsby *only* has a dev mode so what would normally be a dev dependency (like Babel) is just a regular dependency in Gatsby :)